travis/tests-miniwdl.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -ex -o pipefail
+
+mkdir -p workflows
+cp -r test workflows/
+cd workflows
+
+miniwdl run_self_test
+
+for workflow in ../pipes/WDL/workflows/*.wdl; do
+	workflow_name=$(basename $workflow .wdl)
+	input_json="test/input/WDL/test_inputs-$workflow_name-local.json"
+	expected_output_json="test/input/WDL/test_outputs-$workflow_name-local.json"
+	if [ -f $input_json ]; then
+		date
+		echo "Executing $workflow_name using miniWDL on local instance"
+		miniwdl run -i $input_json -d $workflow_name/. --error-json $workflow
+		if [ -f $workflow_name/outputs.json ]; then
+			echo "$workflow_name SUCCESS -- outputs:"
+			cat $workflow_name/outputs.json
+			if [ -f $expected_output_json ]; then
+				echo "$workflow_name -- validating outputs"
+				touch expected actual
+				for k in `cat $expected_output_json | jq -r 'keys[]'`; do
+					echo -n "$k=" >> expected
+					echo -n "$k=" >> actual
+					cat $expected_output_json       | jq -r '.["'$k'"]' >> expected
+					cat $workflow_name/outputs.json | jq -r '.["'$k'"]' >> actual
+				done
+				diff expected actual
```

Comment:
Maybe add `--brief` (Output only whether files differ), `--text` (Treat all files as text)?

Reply:
So the output will always be text anyway, and I actually kind of want the full diff. An earlier version of this had the `diff` inside the for loop. The for loop loops through every key/val pair in the expected_output_json and writes `key=val\n` on every line of the `expected` and `actual` text files. `diff` returns a non-zero exit code if there's a difference, so it used to abort at the first difference -- now (in this version) it aborts after checking all the expected key/val pairs for a workflow and shows all differences (`--brief` would lose that)